veneer/percona-server-mongodb-operator.yaml

```diff
@@ -4,7 +4,9 @@ GenerateMinorChannels: false
 Fast:
   Bundles:
   - Image: quay.io/operatorhubio/percona-server-mongodb-operator:v1.15.0
+  - Image: docker.io/perconalab/percona-server-mongodb-operator:1.17.0-community-bundle
 Stable:
   Bundles:
   - Image: quay.io/operatorhubio/percona-server-mongodb-operator:v1.14.0
```

Comment:
@oksana-grishchenko shouldn't we remove this version? We agreed that every everest version would only support 2 versions of a given operator.
Also, upgrading from everest 1.1.1 to 1.2.0 should be blocked if the users are running psmdbo v1.14.0

Reply:
@recharte 
> We agreed that every everest version would only support 2 versions of a given operator.
lost it. We should update it, thank you. Also, we need to update the 1.17.0 to point out to percona repo instead of perconalab, the official image is going to be available today

Reply:
> Also, upgrading from everest 1.1.1 to 1.2.0 should be blocked if the users are running psmdbo v1.14.0

is there a mechanism to do so or we need to implement it?

Reply:
> is there a mechanism to do so or we need to implement it?

I think that this should already been in place because I remember discussing it as a requirement but to be honest I can't remember if it got implemented or not, we should check.

Reply:
The mechanism exists AFAIK.

We need to update the metadata in VS like so:

```yaml
version: 1.2.0-rc1
supported:
  cli: '>= 1.2.0'
  psmdbOperator: '> 1.14.0'
```